### ISSUE TITLE
Please include Windows binaries in releases!

### ISSUE THREAD

**Issue by chewbacca9000** [envoyproxy/envoy#24711]
Currently, your releases include Linux binaries (which is an improvement, because before there were no binaries).  However, the Windows binaries are still not included.  This is a huge pain.  Can we please include the Windows binaries in the releases for new releases?

**Comment by snowp** [comment]
@phlax @wrowe Can either of you speak to the state of Windows releases?

**Comment by phlax** [comment]
yes - its planned  but there are a couple of hurdles - in particular signing the binaries

i would encourage anyone interested to join the #envoy-release slack channel as that is where most of the conversation around this is going on
